From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: LuminolMC CI <ci@luminolmc.com>
Date: Thu, 17 Jul 2025 07:35:53 +0800
Subject: [PATCH] SpeedTrain: Remove hardcoded minecart speed limits


diff --git a/net/minecraft/world/entity/vehicle/Minecart.java b/net/minecraft/world/entity/vehicle/Minecart.java
index dea43b8027a63da48d5c45d180515093cdd2d331..b801f3713f3e502b349b7ad4a1cb88aaef277ec9 100644
--- a/net/minecraft/world/entity/vehicle/Minecart.java
+++ b/net/minecraft/world/entity/vehicle/Minecart.java
@@ -103,4 +103,4 @@ public class Minecart extends AbstractMinecart {
             this.playerRotationOffset = f;
         }
     }
-}:
\ No newline at end of file
+}
\ No newline at end of file
